site/src/pages/get-involved.mdx

```diff
@@ -14,9 +14,18 @@ The best place to start is by participating in one or more of our communication
 
 - [Discord](https://discord.gg/DEWjhSw): Ask a question, learn about what's happening, be alerted of upcoming meetings and opportunities to get involved in the Open UI community.
 - [Open UI GitHub](https://github.com/openui/open-ui): Our repo where we track issues, progress on spec proposals, and publish our research for public consumption
-- [Open UI Telecon Agendas](https://github.com/openui/open-ui/tree/main/meetings/telecon): We have weekly telecons at 11:00 AM Pacific Standard Time, you can find previous minutes and upcoming agendas here. If you want to get added to the calendar invite contact [Greg Whitworth](https://twitter.com/gregwhitworth) or [Una Kravets](https://twitter.com/Una) on [Discord](https://discord.com/invite/DEWjhSw) or [Twitter](https://twitter.com/openuicg?lang=en)
 - [Open UI community group](https://www.w3.org/community/open-ui/): In order to add substantial contributions to Open UI you need to join the Open UI CG
 
+## Joining the Open UI Telecon
+
+We have a weekly telecon on Thursdays at 11AM - 12PM PST. We post the agenda will be posted by Tuesday EOD on the Discord telecon channel along with the Jitsi video telecon link. The agenda and minute archives can be found [here](https://github.com/openui/open-ui/tree/main/meetings/telecon).
+
+## Open UI IRC Channel
+
+Like many working or community groups in the W3C we leverage IRC. We primarily use it during the telecons to leverage the various bots that have been created through the years for drafting minutes and commenting on related Github bots. Our IRC channel is #openui and you can use the free IRC server on W3C at [https://irc.w3.org/](https://irc.w3.org/) if you don't have a preferred client.
+
+For useful and common IRC commands, please see this [guide](https://www.w3.org/wiki/IRC). 
```

Comment:
It might be worth lifting a few more IRC-related links from https://wiki.csswg.org/tools#meeting-tools , or maybe just linking to it.